smm/README.md

```diff
@@ -55,8 +55,18 @@ Represent the relationship between Measure categories, and other Measure element
 **TimeStamp Primitive Type**
 This primitive represents a point in time, such as the combination of a date and a time within a day.
 ## Extensions
+SMM provides a simple extension mechanisms that provide a uniform meta-model pattern for extending SMM. It allows ad-hoc users attributes and annotations definitions to instances of SMM Elements. 
```

Comment:
+SMM provides a simple extension mechanism that provide a uniform meta-model pattern for extending SMM. 
_It allows ad-hoc user attributes and annotations definitions to instances of SMM Elements._ (missing verb in the sentence: allows users to do what?)